fcrepo-jms/src/main/java/org/fcrepo/jms/headers/DefaultMessageFactory.java

```diff
@@ -116,6 +120,32 @@ public Message getMessage(final FedoraEvent jcrEvent,
         return message;
     }
 
+    /**
+     *
+     * @param fixityEvent
+     * @param jmsSession
+     * @return
+     * @throws RepositoryException
+     * @throws JMSException
+     */
+
+    public Message getFixityMessage(final FixityEvent fixityEvent,
```

Comment:
If this is a new part of the public contract for a message factory, it needs to be in `JMSEventMessageFactory`, not introduced here.